src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/CombinationItemType.php

```diff
@@ -55,65 +58,136 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
+                'attr' => [
+                    'class' => 'combination-impact-on-price-tax-included',
+                    'data-order-by' => 'price',
+                ],
+                'label' => $this->trans('Impact on price (tax incl.)', 'Admin.Catalog.Feature'),
             ])
-            ->add('impact_on_price', SubmittableInputType::class, [
-                'type' => MoneyType::class,
+            ->add('final_price_te', TextPreviewType::class, [
                 'attr' => [
-                    'class' => 'combination-impact-on-price',
+                    'class' => 'combination-final-price',
+                    'data-order-by' => 'price',
                 ],
+                'label' => $this->trans('Final price (tax excl.)', 'Admin.Catalog.Feature'),
+            ])
+            ->add('delta_quantity', DeltaQuantityType::class, [
+                'attr' => [
+                    'data-order-by' => 'quantity',
+                ],
+                'delta_label' => false,
+                'label' => $this->trans('Quantity', 'Admin.Global'),
             ])
-            ->add('final_price_te', HiddenType::class)
-            ->add('delta_quantity', SubmittableDeltaQuantityType::class)
             ->add('is_default', RadioType::class, [
                 'label' => false,
                 'attr' => [
                     'class' => 'combination-is-default-input',
                 ],
             ])
-            ->add('edit', IconButtonType::class, [
-                'icon' => 'mode_edit',
-                'attr' => [
-                    'class' => 'edit-combination-item tooltip-link',
-                    'data-toggle' => 'pstooltip',
-                    'data-original-title' => $this->trans('Edit', 'Admin.Global'),
+            ->add('actions', ButtonCollectionType::class, [
+                'buttons' => [
+                    'edit' => [
+                        'type' => IconButtonType::class,
+                        'options' => [
+                            'icon' => 'mode_edit',
+                            'attr' => [
+                                'class' => 'edit-combination-item tooltip-link',
+                                'data-toggle' => 'pstooltip',
+                                'data-original-title' => $this->trans('Edit', 'Admin.Global'),
+                            ],
+                        ],
+                    ],
+                    'delete' => [
+                        'type' => IconButtonType::class,
+                        'options' => [
+                            'icon' => 'delete',
+                            'attr' => [
+                                'class' => 'delete-combination-item tooltip-link',
+                                'data-modal-title' => $this->trans('Delete item', 'Admin.Notifications.Warning'),
+                                'data-modal-message' => $this->trans('Are you sure you want to delete this item?', 'Admin.Notifications.Warning'),
```

Comment:
```suggestion
                                'data-modal-message' => $this->trans('Delete selected item?', 'Admin.Notifications.Warning'),
```

Reply:
Ok, I'm surprised because all the other delete buttons on the product page use this wording for now:

https://github.com/PrestaShop/PrestaShop/blob/b877339a9e38e0e12a74a17e6a446d329453c75b/src/PrestaShopBundle/Form/Admin/Sell/Product/FooterType.php#L113
https://github.com/PrestaShop/PrestaShop/blob/f27c365dfcaba7c5700afda905f5013fa0666eeb/src/PrestaShopBundle/Form/Admin/Sell/Product/Options/CustomizationFieldType.php#L91
https://github.com/PrestaShop/PrestaShop/blob/cdbd6d08510230839a66e55195cb582093a3882a/src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/FeatureValueType.php#L114

Does it mean they should all be fixed?

And in the grid we use this trait which is common to almost all grids and it uses a similar wording:

https://github.com/PrestaShop/PrestaShop/blob/2dce139eeb7ea4f070af91b84b0428650b0ae504/src/Core/Grid/Definition/Factory/DeleteActionTrait.php#L53

(In this case I'm think the plural form may not be needed since this action is always on a single element from the row)

Reply:
I know :( But "Are you sure" is really not a best practice in UX writing. It makes the user hesitate, it's not easily scannable. I think we should go straight to the point. Sometimes it can be useful to make the user hesitate a bit. For example, the activation of an experimental feature could lead to data loss if done incorrectly. So, it's ok to make the user hesitate by starting the sentence with "Are you sure". If you think this should be done in another PR I have no problem with that.